test/e2e/configs.go

```diff
@@ -18,6 +18,7 @@ type testType string
 
 const (
 	metrics     testType = "metrics"
+	rules 		testType = "rules"
```

Comment:
`make format` should fix these rogue whitespaces :+1:

Reply:
weirdly if I run `make format` locally I get:
```bash
jessica@cupsofwonder:~/workspace/api|rules-add-e2e ⇒  make format
/home/jessica/go/bin/golangci-lint-v1.21.0 run --fix --enable-all -c .golangci.yml
WARN [runner/nolint] Found unknown linters in //nolint directives: cyclop, exhaustivestruct, goerr113, gomnd, intefacer, paralleltest, testpackage 
```
so somehow it did not give any more output than that and also did not change any of the files.

I then tried to run with my local `golangci-lint` (with the latest version - 1.43.0) and it found some issues in the code, so I think we may tackle this separately? I wonder if would be good to also update the `golangci-lint` version (with the latest version I get also some warnings of linters that are deprecated and we use them in the `nolinter` directive (e.g. `WARN [runner/nolint] Found unknown linters in //nolint directives: intefacer`)

Should I create a separate issue for that? Maybe makes sense to also link to this issue @matej-g already opened: https://github.com/observatorium/api/issues/212
For now I run manually `gofmt` to this config file, so the whitespaces should be fixed

Reply:
Also, shouldn't the lint step fail?

Reply:
Seems to be fixed now :shrug:

Reply:
you mean the whitespaces or `make format`?
To fix the whitespaces I ran `gofmt` manually, but I thought would be fixed by running `make format` instead (which does not seem to be the case) :(